### PR TITLE
Apps doc updates

### PR DESCRIPTION
**Pull request by andamian**

### CODE REVIEW

**Comment by andamian** [comment]
@molinaro-m - this looks better. Thanks for the tip.

**Review by molinaro-m — APPROVED** [review]
Perfect! Now it works, both locally and on the webtest preview branch

**Comment by molinaro-m** [comment]
> @molinaro-m - this looks better. Thanks for the tip.

it's more like what we will operate tomorrow.
I think the fork+PR will need to be the forward mechanism when this goes production.

**Review by JeremyMcCormick — CHANGES_REQUESTED** [review]
Conflict on `content/_index.html` needs to be fixed. There was a big update to the site style that you probably don't have integrated into your branch.

Second, more minor issue, and not to be too picky about Git stuff since we haven't broadcasted this widely yet, but `apps` is not a very good branch name. What we are preferring is making an issue in the tracker first, then using it in the name like `iss1234` or `iss1234-port-apps`. I won't block the PR based on this (just the conflict needs to be fixed), but please in the future make an issue first and then name your branch after it.

**Comment by gmantele** [comment]
This conflict is because of my last commit. All the links are now in a different file: `layout/partials/sitemap.html`. This way, it can be put at the bottom of all pages.

File: content/pages/astronomers/applications.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---
+title: "VO Applications"
+date: 2024-02-09T10:00:00+02:00
```
Comment (JeremyMcCormick) [inline content/pages/astronomers/applications.md:3]: The `date` can be left out - this is not correct anyways and Hugo will insert it automatically.

**Comment by gmantele** [comment]
I can, indeed. I will probably wait for the PR #23 to be merged first, as it touches as well the `sitemap.html` file in the way it is done in this PR.

**Comment by gmantele** [comment]
By the way, should this PR stay a Draft?

**Comment by andamian** [comment]
@gmantele and @JeremyMcCormick - thanks for the feedback. This is a draft PR that I've started in preparation for the hackathon. I can probably solve the conflict at the end when the PR is ready which right now is not.

**Comment by JeremyMcCormick** [comment]
> @gmantele and @JeremyMcCormick - thanks for the feedback. This is a draft PR that I've started in preparation for the hackathon. I can probably solve the conflict at the end when the PR is ready which right now is not. The page Apps page has not been worked on.

Understood, we can look at this tomorrow then. 👍

**Comment by andamian** [comment]
@JeremyMcCormick - I'm probably jumping the gun here without having clear directions on how we are going to proceed. I just assume that in this PR we will fix all the issues raised in https://docs.google.com/spreadsheets/d/15TE9LCNdDyBhAQF_TyX2AqXSXW8RxQqHkU_bBKxQK7I/edit#gid=0. I will adapt later if that's not the case.

**Comment by JeremyMcCormick** [comment]
> @JeremyMcCormick - I'm probably jumping the gun here without having clear directions on how we are going to proceed. I just assumed that in this PR we will fix all the issues raised in https://docs.google.com/spreadsheets/d/15TE9LCNdDyBhAQF_TyX2AqXSXW8RxQqHkU_bBKxQK7I/edit#gid=0. I will adapt later if that's not the case.

Yes, we are hoping to fix at least some of those issues.

Please read the contribution guide:

https://github.com/ivoa/ivoa-web/blob/main/CONTRIBUTING.md

It may clear some things for you in terms of expeceted developer workflow.

**Comment by gmantele** [comment]
Now that the main conflicting changes are done, maybe we should try to fix the conflict in this branch so that the editing team can work with no problem this afternoon.

**Comment by JeremyMcCormick** [comment]
> Now that the main conflicting changes are done, maybe we should try to fix the conflict in this branch so that the editing team can work with no problem this afternoon.

Can we put this on a branch that references an issue in the tracker and then reopen a new PR?

I don't want to get in the habit of accepting vague branch names like this, as it is an organizational mess. The issue page also provides a nice place for discussion on the issue.

@andamian You can just branch from your existing one like `git checkout apps && git checkout -b iss1234` which will keep your work.

**Comment by andamian** [comment]
Replaced with #26

**Pull request closed without merging**